### PR TITLE
Testing of "core" package

### PR DESCRIPTION
**Pull request by samechelon**

### CODE REVIEW

**Comment by leogr** [comment]
@leodido add your review pls

File: schemas/core/common_test.go [hunk 1]
```diff
@@ -61,35 +61,27 @@ func TestCommonValidation(t *testing.T) {
...
 		require.IsType(t, (validator.ValidationErrors)(nil), errors)
 	}
-	require.Len(t, errors, 1)
 
-	// Empty Schema fail on gt validation tag, ID fails on excludes validation tag
-	c = CommonAttributes{
-		Schemas: []string{},
-		ID:      "bulkId",
-	}
+	// Wrong URN fail on urn validation tag, right ID
```
Comment (leodido) [inline schemas/core/common_test.go]: Ok, please improve comment writing `valid ID`. It is more readable and understandable.

File: schemas/core/common_test.go [hunk 2]
```diff
@@ -61,35 +61,27 @@ func TestCommonValidation(t *testing.T) {
...
 	errors = validation.Validator.StructExcept(c, "Meta")
-	for _, err := range errors.(validator.ValidationErrors) {
-		require.NotNil(t, err)
-		require.IsType(t, (validator.ValidationErrors)(nil), errors)
-	}
-	require.Len(t, errors, 2)
+	require.Error(t, errors)
+	require.IsType(t, (validator.ValidationErrors)(nil), errors)
 
 	// Right URN and valid ID
```
Comment (leodido) [inline schemas/core/common_test.go]: Comment with `both schemas contents (ie., URNs) and ID valid`

File: schemas/core/resource_type.go [hunk 3]
```diff
@@ -13,7 +13,7 @@ type ResourceType struct {
 	Endpoint         string            `json:"endpoint" validate:"startswith=/,required"`
 	Description      string            `json:"description,omitempty"`
 	Schema           string            `json:"schema" validate:"urn,required"`
-	SchemaExtensions []SchemaExtension `json:"schemaExtensions,omitempty"`
+	SchemaExtensions []SchemaExtension `json:"schemaExtensions,omitempty" validate:"dive"`
```
Comment (leodido) [inline schemas/core/resource_type.go:16]: :+1:  :smirk:

**Review by leodido — COMMENTED** [review]
Except some comments/descriptions it LGTM.

File: schemas/core/resource_type.go (hunk 3 above)
Comment (leodido) [reply schemas/core/resource_type.go:16]: Bravo.

**Pull request merged**